Rubberduck.Core/UI/UnitTesting/TestExplorerViewModel.cs

```diff
@@ -341,6 +369,28 @@ private void ExecuteResetResultsCommand(object parameter)
             Tests.Refresh();
         }
 
+        private void ExecuteIgnoreTestCommand(object parameter)
+        {
+            var rewriteSession = RewritingManager.CheckOutCodePaneSession();
+            var qualifiedContext = new QualifiedContext(_mousedOverTestMethod.Declaration.QualifiedModuleName, _mousedOverTestMethod.Declaration.Context);
```

Comment:
Why not use the overload of `AddAnnotation` taking a declaration?

Reply:
I derped when I was coding and didn't see it...